### PR TITLE
Move some console related code from debugger and inspector screens to shared/console.

### PR DESCRIPTION
**Pull request by polina-c**
RELEASE_NOTE_EXCEPTION=[no functional changes]

### CODE REVIEW

File: packages/devtools_app/lib/src/shared/console/eval/inspector_service.dart
Comment (jacob314) [inline packages/devtools_app/lib/src/shared/console/eval/inspector_service.dart:15]: This has a lot of non-console stuff in it. I would suggest either leaving it in the inspector and accepting slightly tangled imports or move it to a shared directory. Seems odd to have this core inspector logic under the console rather than inspector or shared.

File: packages/devtools_app/lib/src/shared/console/widgets/description.dart
Comment (jacob314) [inline packages/devtools_app/lib/src/shared/console/widgets/description.dart:258]: this seems a bit inverted. these are much more inspector specific styles than console styles. If you want a neutral name, these could be diagnostics_text_styles.

File: packages/devtools_app/lib/src/shared/console/eval/diagnostics_node.dart
Comment (jacob314) [inline packages/devtools_app/lib/src/shared/console/eval/diagnostics_node.dart:2]: This also isn't a console thing. If you want to move it, it should go in shared not console.

File: packages/devtools_app/lib/src/shared/console/primitives/text_styles.dart
Comment (jacob314) [inline packages/devtools_app/lib/src/shared/console/primitives/text_styles.dart:9]: why not continue the previous pattern of top level members imported with a prefix? In general if we were to use one pattern I would think we would migrate to that pattern not to this pattern that violates the lints we have enabled.

**Review by jacob314 — APPROVED** [review]
lgtm
LGTM as long as there is a cleanup CL after the 3 chained CLs are landed moving things to a common dir.

File: packages/devtools_app/lib/src/shared/console/primitives/text_styles.dart
Comment (polina-c) [reply packages/devtools_app/lib/src/shared/console/primitives/text_styles.dart:9]: Discussed offline and agreed enum-like static classes are more convenient than top level members.

File: packages/devtools_app/lib/src/shared/console/eval/inspector_service.dart
Comment (polina-c) [reply packages/devtools_app/lib/src/shared/console/eval/inspector_service.dart:15]: Addressed in https://github.com/flutter/devtools/pull/5006.

File: packages/devtools_app/lib/src/shared/console/widgets/description.dart
Comment (polina-c) [reply packages/devtools_app/lib/src/shared/console/widgets/description.dart:258]: Renamed and moved to shared root in #5021

**Pull request merged**
